### PR TITLE
[DBZ-2510]: Add more logs

### PR DESCRIPTION
**Pull request by bingqinzhou**
Log keyspace, table and column when an error occurs.

### CODE REVIEW

File: debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/CommitLogReadHandlerImpl.java [hunk 1]
```diff
@@ -395,23 +395,30 @@ private void handleRowModifications(Row row, RowType rowType, PartitionUpdate pu
...
-            CellData cellData = new CellData(name, value, null, CellData.ColumnType.PARTITION);
-            after.addCell(cellData);
+            try {
+                String name = cd.name.toString();
+                Object value = partitionKeys.get(cd.position());
+                CellData cellData = new CellData(name, value, null, CellData.ColumnType.PARTITION);
+                after.addCell(cellData);
+            }
+            catch (Exception e) {
+                LOGGER.error("Failed to populate Column {} with Type {} of Table {} in KeySpace {}.",
```
Comment (gunnarmorling) [inline debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/CommitLogReadHandlerImpl.java:405]: Let's better either log _or_ rethrow, but not both at the same time, as it'll clutter log files. So I'd suggest to wrap the original exception in a `DebeziumException`, adding the message with the additional context you'd like to propagate. It'll all be logged at once then.

**Review by gunnarmorling — CHANGES_REQUESTED** [review]
Thanks, @bingqinzhou! One comment inline which applies to all the changed code parts essentially. Thanks!

**Comment by gunnarmorling** [comment]
@bingqinzhou, LGTM now. Could you only squash everything into one commit and rebase it to the current master (in general, let's work with rebases over merges for the sake of a linear and thus cleaner history)? Thanks!

**Comment by gunnarmorling** [comment]
@bingqinzhou, one more question actually: why is DBZ-2498 reverted here?

**Comment by bingqinzhou** [comment]
> @bingqinzhou, one more question actually: why is DBZ-2498 reverted here?

Not sure, it seems that 462a5f4, 869b513, 8efe8bc, 7428150 are actually commits from #190 instead of this one. The commit history in my local master seems be messed up, I'm deleting this PR and created #193 with a cleaner commit history.

**Comment by gunnarmorling** [comment]
Gotcha, no worries. Merged #193 and will close this one. Thanks, @bingqinzhou!

**Pull request closed without merging**